src/middleware/jiraSymmetricJwtMiddleware.test.ts

```diff
@@ -0,0 +1,170 @@
+import { encodeSymmetric } from "atlassian-jwt";
+import { when } from "jest-when";
+import { booleanFlag, BooleanFlags } from "~/src/config/feature-flags";
+import { getLogger } from "~/src/config/logger";
+import { jiraSymmetricJwtMiddleware } from "~/src/middleware/jiraSymmetricJwtMiddleware";
+import { Installation } from "~/src/models/installation";
+
+
+const logger = getLogger("jira-jwt-verify-middleware.test");
+jest.mock("models/installation");
+jest.mock("config/feature-flags");
+const testSharedSecret = "test-secret";
+
+describe("jiraSymmetricJwtMiddleware", () => {
+	let res;
+	let next;
+	let installation;
+
+	beforeEach(async () => {
+		res = {
```

Comment:
also, instead of mocking everything, would it make sense to create the app? This way we could test `req.path` stuff, too, couldn't we? (We might make a mistake during mocking..)  Like here:
http://github.com/atlassian/github-for-jira/blob/1fc1bfb87eb30b4a805aa9990461b203d4825575/src/routes/api/api-router.test.ts#L19-L19